src/Differ/UnifiedDiffer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Differ;
+
+use PhpCsFixer\Diff\GeckoPackages\DiffOutputBuilder\UnifiedDiffOutputBuilder;
+use PhpCsFixer\Diff\v2_0\Differ;
+
+/**
+ * @author SpacePossum
+ */
+final class UnifiedDiffer implements DifferInterface
+{
+    /**
+     * @var Differ
+     */
+    private $differ;
+
+    public function __construct()
+    {
+        $this->differ = new Differ(new UnifiedDiffOutputBuilder([
+            'fromFile' => 'Original', // TODO
```

Comment:
What is left to do here?

Reply:
ideal case would be that the `diff` contains the file name (and not 'Original')